src/data/markdown/docs/30 xk6-browser/01 Getting started/02 Installation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Installation'
+excerpt: 'A step-by-step guide on how to install xk6-browser.'
+---
+
+xk6-browser is currently being developed as a [k6 extension](/extensions). You have to run a k6 version built with the browser extension to use the [browser-level APIs](#browser-level-apis) in your k6 tests.
+
+### Download a release binary
+
+The quickest way to get started is to [download a release binary from GitHub](https://github.com/grafana/xk6-browser/releases).
+
+This option is recommended if you don't want to build your own binary with [xk6](https://github.com/grafana/xk6), which can be challenging in some cases.
+
+### Build from source
+
+If you want to get the latest changes of the xk6-browser extension, you can also build a binary from source.
+
+To build a binary with the extension:
+1. Ensure you have [Go](https://golang.org/doc/install) and [Git](https://git-scm.com/) installed.
+2. Run the following commands in a terminal:
+
+```bash
+# Install xk6
+go install go.k6.io/xk6/cmd/xk6@latest
+
+# Build the xk6-browser binary
+xk6 build --output xk6-browser --with github.com/grafana/xk6-browser
```

Comment:
On Windows this currently needs to be `xk6 build --output xk6-browser.exe --with github.com/grafana/xk6-browser` (NOTE: the `.exe` extension).

Would be good if we had Windows specific instruction here, like we do [here](https://k6.io/docs/getting-started/running-k6/#adding-more-vus).

Reply:
We have an open ticket for this, so maybe it's worth doing it separately? https://github.com/grafana/xk6-browser/issues/592

Reply:
Also, would it be worth suggesting that the user uses `xk6 build --output xk6-browser --with github.com/grafana/xk6-browser@latest` which will be the up to date version on the `main` branch, vs `xk6 build --output xk6-browser --with github.com/grafana/xk6-browser` which will build the last tagged version?

Reply:
I don't mind adding the same instructions on Windows in this PR as there are still outstanding changes required from our external design team.

Regarding `xk6 build --output xk6-browser --with github.com/grafana/xk6-browser@latest`, I asked about this before during one of our team catch-ups and I think the reply was we want our users to use the latest release version because it's the stable version. @imiric WDYT?

Reply:
@ankur22 I don't think that's right.

If `@latest` is not specified, xk6 will append it, so whether the user specifies it or not doesn't matter. And `@latest` will build the latest released/tagged version, not the latest `main` commit. This is why we often suggest internal testers and users who want the latest _unstable_ version to install with `@main` instead. So leaving this as is is fine. I don't think we even need to document the `@main` command, just to avoid confusing users more.

As for appending `.exe` on Windows, I agree and think we should mention it here, and anywhere else we document `xk6 build`.

Reply:
@imiric thanks for clarifying 👍

Reply:
Addressed as part of https://github.com/grafana/k6-docs/pull/828/commits/9aafb34800a1dad3e764367391e28451ad0b6bd2